Code/GraphMol/Wrap/MolOps.cpp

```diff
@@ -1651,6 +1661,40 @@ to the terminal dummy atoms.\n\
     python::def("AssignRadicals", assignRadicalsMol, (python::arg("mol")),
                 docString.c_str());
 
+    docString =
```

Comment:
Just as a reference for future PRs: now that C++ supports multi-line strings, this horrible mess with `\n\` is no longer necessary

Reply:
this change wasn't exactly what I meant.
Now this is all one line of text, which is not going to look good when it's output.
I was talking about doing something like `R"DOC( multi_line_text_with_returns )DOC";`
Either that or the `\n\` version would be fine